doap_Kafka.rdf

```diff
@@ -13,30 +13,30 @@
     (the "License"); you may not use this file except in compliance with
     the License.  You may obtain a copy of the License at
 
-         http://www.apache.org/licenses/LICENSE-2.0
+         https://www.apache.org/licenses/LICENSE-2.0
 
     Unless required by applicable law or agreed to in writing, software
     distributed under the License is distributed on an "AS IS" BASIS,
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
 -->
-  <Project rdf:about="http://kafka.apache.org/">
+  <Project rdf:about="https://kafka.apache.org/">
     <created>2014-04-12</created>
     <license rdf:resource="http://usefulinc.com/doap/licenses/asl20" />
     <name>Apache Kafka</name>
-    <homepage rdf:resource="http://kafka.apache.org/" />
-    <asfext:pmc rdf:resource="http://kafka.apache.org" />
+    <homepage rdf:resource="https://kafka.apache.org/" />
+    <asfext:pmc rdf:resource="https://kafka.apache.org" />
     <shortdesc>Apache Kafka is a distributed, fault tolerant, publish-subscribe messaging.</shortdesc>
     <description>A single Kafka broker can handle hundreds of megabytes of reads and writes per second from thousands of clients. Kafka is designed to allow a single cluster to serve as the central data backbone for a large organization. It can be elastically and transparently expanded without downtime. Data streams are partitioned and spread over a cluster of machines to allow data streams larger than the capability of any single machine and to allow clusters of co-ordinated consumers. Kafka has a modern cluster-centric design that offers strong durability and fault-tolerance guarantees. Messages are persisted on disk and replicated within the cluster to prevent data loss. Each broker can handle terabytes of messages without performance impact.</description>
     <bug-database rdf:resource="https://issues.apache.org/jira/browse/KAFKA" />
-    <mailing-list rdf:resource="http://kafka.apache.org/contact.html" />
-    <download-page rdf:resource="http://kafka.apache.org/downloads.html" />
+    <mailing-list rdf:resource="https://kafka.apache.org/contact.html" />
+    <download-page rdf:resource="https://kafka.apache.org/downloads.html" />
     <programming-language>Scala</programming-language>
-    <category rdf:resource="http://projects.apache.org/category/big-data" />
+    <category rdf:resource="https://projects.apache.org/category/big-data" />
```

Comment:
nit: looks like current url in not working.  New url: https://projects.apache.org/projects.html?category#big-data